src/hypothesis/strategies.py

```diff
@@ -624,19 +624,30 @@ def streaming(elements):
 @defines_strategy
 def characters(whitelist_categories=None, blacklist_categories=None,
                blacklist_characters=None, min_codepoint=None,
-               max_codepoint=None):
+               max_codepoint=None, whitelist_characters=None):
     """Generates unicode text type (unicode on python 2, str on python 3)
     characters following specified filtering rules.
 
-    This strategy accepts lists of Unicode categories, characters of which
-    should (`whitelist_categories`) or should not (`blacklist_categories`)
-    be produced.
+    When no filtering rules are specifed, any character can be produced.
 
-    Also there could be applied limitation by minimal and maximal produced
-    code point of the characters.
+    If `min_codepoint` or `max_codepoint` is specifed, then only characters
+    having a codepoint in that range will be produced.
 
-    If you know what exactly characters you don't want to be produced,
-    pass them with `blacklist_characters` argument.
+    If `whitelist_categories` is specified, then only characters from those
```

Comment:
Looks like you intended argument names to be literals, right?  This is one of the annoying differences between markdown and restructured text: for Sphinx, you need double backticks for an inline literal.

(single backticks are instead a magic cross-reference, so this could eventually hyperlink to something completely unrelated if it has the right identifier.)